Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt

```diff
@@ -346,8 +347,8 @@ private fun PhotoQuestion(
                     contentDescription = null,
                     modifier = Modifier
                         .fillMaxWidth()
-                        // We set a minimum height, so that Coil has a height to initially use
-                        .heightIn(min = 96.dp),
+                        .heightIn(96.dp)
+                        .aspectRatio(1.22f)
```

Comment:
`1.22f` is an interesting aspect ratio. I'd probably use `aspectRatio(4 / 3f)` which is more typical and common.

Reply:
Figma design has an image size of 371 / 302 px that is why I chose that aspect ratio, but I will update to use 4/3f